docs/root/intro/version_history.rst

```diff
@@ -58,6 +58,8 @@ Version history
   restart Envoy. The behavior will not switch until the connection pools are recreated. The new
   circuit breaker behavior is described :ref:`here <arch_overview_circuit_break>`.
 * upstream: changed load distribution algorithm when all priorities enter :ref:`panic mode<arch_overview_load_balancing_panic_threshold>`.
+* AWS Lambda: added :ref:`AWS Lambda filter <config_http_filters_aws_lambda>` that converts HTTP requests to Lambda invokes. This effectively makes Envoy act as an egress
```

Comment:
Should probably say `aws_lambda: added ...` and move this up to the top since these are now alphabetical.